pkg/vm/builtins/layouts.go

```diff
@@ -37,6 +37,15 @@ func getPlainLayout() Layout {
 	return Layout{Name: "plain", RcUnits: 16, Builtins: []LayoutBuiltin{}}
 }
 
+func getRecursiveLayout() Layout {
+	return Layout{Name: "recursive", RcUnits: 16, Builtins: []LayoutBuiltin{
```

Comment:
`RcUnits` seems to be `4` in the lambdaclass vm 🤔

Reply:
true! my bad, will update this